app/starship/styles.css

```
@@ -0,0 +1,1097 @@
+    cursor: pointer;
+    background-color: rgba(6, 6, 6, 0.007);
+}
+
+.sidebar-header-button:active {
+    cursor: pointer;
+    background-color: rgba(6, 6, 6, 0.007);
+}
+
+* {
+    -webkit-tap-highlight-color: transparent;
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

-webkit-tap-highlight-color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)